src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.security.SecureRandom;
+
+public class FigureSupplier extends ColorSupplier {
+    private static final int FIGURE_COUNT = 5;
+    private final SecureRandom random = new SecureRandom();
+
+    public Figure getRandomFigure() {
+        int figureNumber = random.nextInt(FIGURE_COUNT) + 1;
+        int randomSide = random.nextInt(100);
+        int randomSide1 = random.nextInt(100);
+        int randomSide2 = random.nextInt(100);
+        int randomRadius = random.nextInt(100);
```

Comment:
100 is a magic number - should be constant

also, maybe you can create helper private method `getRandomNumber()` here? and call it when you create figures?